src/Router/Router.ts

```diff
@@ -33,13 +33,13 @@ export default class Router {
     this.transitions[TxStatus.Executed] = this.executed.bind(this);
     this.transitions[TxStatus.Missed] = (txRequest: any) => {
       console.log('missed: ', txRequest.address);
-      this.config.cache.del(txRequest.address);
+      // this.config.cache.del(txRequest.address);
       return TxStatus.Missed;
     };
 
     this.transitions[TxStatus.Done] = (txRequest: any) => {
       console.log('done: ', txRequest.address);
-      this.config.cache.del(txRequest.address);
+      // this.config.cache.del(txRequest.address);
```

Comment:
This is not working yet?

Reply:
I recommend uncommenting these two as what's the reason to keep Done tx in memory?

Maybe it's worth to keep missed to run some cleanups, cancel.